recipe/meta.yaml

```
@@ -6,22 +6,22 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/pyportfolioopt-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name}}-{{ version }}.tar.gz
   sha256: 7e94f41c84fb5865c7a64de995a3ba580188f3ba494f6dfbc02721b5de323f6e
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 1
+  number: 0
+  skip: true  # [py<39]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - pip
-    - poetry
-    - python {{ python_min }}
+    - python
+    - poetry-core
```

```suggestion
    - poetry-core >=1.0.0
```
https://github.com/robertmartin8/PyPortfolioOpt/blob/v1.5.6/pyproject.toml#L59